### ISSUE TITLE
Fix telegram/instgram hover color

### ISSUE THREAD

**Issue by orabe** [Informathek/informathek.github.io#5]

**Comment by orabe** [comment]
fixed using: https://fontawesome.com/search?q=instagram&o=r

example of usage: 

```html
<li class="social-item telegram">
 <a target="_blank" rel="noopener noreferrer" href="https://t.me/infor_mathe_k"
  class="social-link telegram">
  <i class="fa-brands fa-telegram"></i> 
 </a>
 </li>
```

**Comment by orabe** [comment]
fixed with this commit: 8050282b9054236c4b4aeb8a34c5e88425be4afd
